### DISCUSSION TITLE
[@types/cheerio] Needs update for a new 1.0.0 release of cheerio

### DISCUSSION THREAD

**Discussion by emvaized** [DefinitelyTyped/DefinitelyTyped#70482] [Issues with a @types package]
cheerio [1.0.0](https://github.com/cheeriojs/cheerio/releases/tag/v1.0.0) introduced new methods in the API: 
- [loadBuffer](https://cheerio.js.org/docs/basics/loading#loadbuffer)
- [stringStream](https://cheerio.js.org/docs/basics/loading#stringstream)
- [decodeStream](https://cheerio.js.org/docs/basics/loading#decodestream)
- [fromURL](https://cheerio.js.org/docs/basics/loading#fromurl)

[a comment by an automated account is not shown]

**Comment by AzSiAz** [comment]
Hello, 

cheerio is now a typescript lib so I don't think there is any need to update `@types/cheerio` for the new version since d.ts appear to be shipped with cheerio itself. If it's not working you can open an issue in the cheerio repo directly

Version prior to typescript move might need to be but those are RC, as far as I checked, so not really sure time should be invested here 😅

**Reply by emvaized** [reply]
Thanks for quick response! It seems that cheerio v1.0.0 no longer exports types, as mentioned in [this issue](https://github.com/cheeriojs/cheerio/issues/3988) and many similar. But I'm not sure if this is a temporary bug, or a breaking change in the new version.

**Reply by emvaized** [reply]
It seems it is a [breaking change](https://github.com/cheeriojs/cheerio/pull/3969) indeed.
